build_projects/shared-build-targets-utils/VersionRepoUpdater.cs

```diff
@@ -60,6 +60,19 @@ public class VersionRepoUpdater
             await UpdateGitHubFile(packageInfoFilePath, packageInfoFileContent, message);
         }
 
+        public async Task PublishVersionBadge(string badgePath, string channel, string pointer)
```

Comment:
See comment above about keeping this file as "repo-agnostic" as possible.  Maybe these could be extension methods on VersionRepoUpdater.